vpr/src/place/place.cpp

```diff
@@ -1409,14 +1427,26 @@ static e_move_result try_swap(float t,
         rlim = std::numeric_limits<float>::infinity();
     }
 
+    if (manual_move) {
+        //pops up the manual move window for the user to input set their move
```

Comment:
I'd move these four lines into a subroutine so this code is as easy to read as possible (readers who don't want manual moves can simply skip over the if and call).